osx/post_installation.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+
+
+# Global Variables
+SCRIPTPATH=$( cd $(dirname $0) ; pwd -P )
+
+KALITE_SHARED="/Users/Shared/kalite"
+PYRUN_NAME="pyrun-2.7"
+PYRUN_DIR="$KALITE_SHARED/$PYRUN_NAME"
+PYRUN="$PYRUN_DIR/bin/pyrun"
+PYRUN_PIP="$PYRUN_DIR/bin/pip"
+BIN_PATH="$PYRUN_DIR/bin"
+ASSESSMENT_SRC="$KALITE_SHARED/assessment/assessment.zip"
+SHEBANGCHECK_PATH="$KALITE_SHARED/scripts/"
+
+SYMLINK_FILE="$KALITE_SHARED/pyrun-2.7/bin/kalite"
+SYMLINK_TO="/usr/local/bin"
+COMMAND_SYMLINK="ln -s $SYMLINK_FILE $SYMLINK_TO"
```

Comment:
You must "force" the symlink to happen if it exists (using `-f` switch).  This may conflict with your `$KALITE` variable below.

What happens when $KALITE="/usr/bin/kalite" and there's also a "/usr/local/bin/kalite" file?